care-pathways-data/ui/src/main/resources/PhenoTips/CarePathwaysDataSearch.xml

```diff
@@ -52,9 +52,9 @@ $!response.setContentType('application/json')##
 #set ($input = $rawInput.toLowerCase())
 #set ($lowercaseResults = [])
 #set ($parentVal = "$!request.parent")
-#set ($standardMarker = "cp:")
+#set ($standardMarker = "CP:")
 #set ($dataClass = "$!{request.dataclass}")
-#foreach($res in $services.query.hql('select distinct datum_prop.value from BaseObject as obj, StringProperty as datum_prop, StringProperty as parent_prop where obj.className = ? and datum_prop.id.id = obj.id and parent_prop.id.id = obj.id and parent_prop.id.name = ? and datum_prop.id.name = ? and parent_prop.value like ? and lower(datum_prop.value) like ? and lower(datum_prop.value) not like ?').bindValues(["$dataClass", "parent", "datum", "%${parentVal}%", "%${input}%", "%${standardMarker}%"]).setLimit(20).execute())
+#foreach($res in $services.query.hql('select distinct datum_prop.value from BaseObject as obj, StringProperty as datum_prop, StringProperty as parent_prop where obj.className = ? and datum_prop.id.id = obj.id and parent_prop.id.id = obj.id and parent_prop.id.name = ? and datum_prop.id.name = ? and parent_prop.value = ? and datum_prop.value like ? and datum_prop.value not like ?').bindValues(["$dataClass", "parent", "datum", "${parentVal}", "%${input}%", "${standardMarker}%"]).setLimit(20).execute())
```

Comment:
`and datum_prop.value like ?` should remain as it was, `and lower(datum_prop.value) like ?`

Reply:
Oops. Fixed